src/services/email.service.js

```diff
@@ -0,0 +1,41 @@
+import 'dotenv/config';
+import nodemailer from "nodemailer";
+
+console.log('program is running');
```

Comment:
The `console.log('program is running');` statement is likely used for debugging purposes. Consider removing it or using a proper logging mechanism if it's intended for production use.